tofu/tests/tests01_geom/test_01_GG.py

```diff
@@ -7,23 +7,15 @@
 import sys
 import numpy as np
 import matplotlib
-matplotlib.use('agg')
 import matplotlib.pyplot as plt
 from matplotlib.path import Path
-import matplotlib.gridspec as mplgrid
-from mpl_toolkits.mplot3d import Axes3D
```

Comment:
Removed because not used, right ? (just to be sure you checked)

Thanks for cleaning-up ! 👍

Reply:
yes :)